kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/OperatorMain.java

```diff
@@ -33,12 +38,14 @@ public class OperatorMain {
     private static final Logger LOGGER = LoggerFactory.getLogger(OperatorMain.class);
     private MeterRegistry registry;
     private final Operator operator;
+    private HttpServer managementServer; // TODO HTTPS support
+    private MetricsHandler metricsHandler;
 
-    public OperatorMain() {
-        this(null);
+    public OperatorMain() throws IOException {
+        this(null, HttpServer.create(new InetSocketAddress("0.0.0.0", 8080), 10));
     }
 
-    public OperatorMain(@Nullable KubernetesClient kubeClient) {
+    public OperatorMain(@Nullable KubernetesClient kubeClient, HttpServer managementServer) {
```

Comment:
NotNull annotation

Reply:
Fixed in 969f6f9447b933eeb83c7e728bb289f7c21641ed